README.md

```diff
@@ -57,37 +57,16 @@ To get the source code from GitHub, type
     cd ./azure-sdk-for-php
 
 > **Note**
-> 
+>
 > The PHP Client Libraries for Microsoft Azure have a dependency on the [HTTP_Request2](http://pear.php.net/package/HTTP_Request2), [Mail_mime](http://pear.php.net/package/Mail_mime), and [Mail_mimeDecode](http://pear.php.net/package/Mail_mimeDecode) PEAR packages. The recommended way to resolve these dependencies is to install them using the [PEAR package manager](http://pear.php.net/manual/en/installation.php).
 
-
 ##Install via Composer
+1. Make sure you have installed **[Compoeser](https://getcomposer.org/doc/00-intro.md#introduction)** in your system.
+It can be installed locally or globally depending on your own needs.
 
-1. Create a file named **composer.json** in the root of your project and add the following code to it:
```

Comment:
You can point any branch or released (tagged) version you want at the constraint, but the preferred way to create a local copy with composer is what I'm proposing; since it is more concise and takes advantage of the command provided for this kind of situations/scencarios.